website/static/js/components/logFeed.js

```diff
@@ -0,0 +1,243 @@
+                    ctrl.paginators().push({
+                        text: i + 1,
+                        url: function() {
+                            ctrl.pageToGet(parseInt(this.text));
+                            if (ctrl.pageToGet() !== ctrl.currentPage()) {
+                                return _buildLogUrl(ctrl.node, ctrl.pageToGet());
+                            }
+                        }
+                    });/* jshint ignore:line */
+                    // function defined inside loop
+                }
+            }
+            // two ellipses
+            else {
+                ctrl.paginators().push({
+                    text: '...',
+                    url: function() { }
+                });
+                for (i = parseInt(ctrl.currentPage()) - 1; i <= parseInt(ctrl.currentPage()) + 1; i++) {
+                    ctrl.paginators().push({
+                        text: i + 1,
+                        url: function() {
+                            ctrl.pageToGet(parseInt(this.text));
+                            if (ctrl.pageToGet() !== ctrl.currentPage()) {
+                                return _buildLogUrl(ctrl.node, ctrl.pageToGet());
+                            }
+                        }
+                    });/* jshint ignore:line */
+                    // function defined inside loop
+                }
+                ctrl.paginators().push({
+                    text: '...',
+                    url: function() { }
+                });
+            }
+            // last page
+            ctrl.paginators().push({
+                text: ctrl.totalPages,
+                url: function() {
+                    ctrl.pageToGet(ctrl.totalPages);
+                    if (ctrl.pageToGet() !== ctrl.currentPage()) {
+                        return _buildLogUrl(ctrl.node, ctrl.pageToGet());
+                    }
+                }
+            });
+            // next page
+            ctrl.paginators().push({
+                url: function() { return ctrl.nextPage(); },
+                text: '>'
+            });
+        }
+
+        return m('.db-activity-list.m-t-md', [
+            // Error message if the log request fails
+            ctrl.failed ? m('p', [
+                'Unable to retrieve logs at this time. Please refresh the page or contact ',
+                m('a', {'href': 'mailto:support@osf.io'}, 'support@osf.io'),
+                ' if the problem persists.'
+            ]) :
+            // Show OSF spinner while there is a pending log request
+            ctrl.logRequestPending() ?  m('.spinner-loading-wrapper', [
+                m('.logo-spin.logo-lg'),
+                m('p.m-t-sm.fg-load-message', 'Loading logs...')
+            ]) :
+            // Display each log item (text and user image)
+            [ctrl.activityLogs() ? ctrl.activityLogs().map(function(item) {
+                var image = m('i.fa.fa-desktop');
+                if (ctrl.node.anonymous) { item.anonymous = true; }
```

Comment:
Why is this necessary? Can you just check `ctrl.node.anonymous` in the line below?